docs/FAQ.rst

```diff
@@ -526,6 +536,9 @@ KECCACK-256
     3", before Keccak was standardized to SHA-3 (FIPS-202) and Keccak has
     minor variations from SHA-3
 
+KME
+    Key Management Enclave Avalon worker. Compare to KME and WPE
```

Comment:
1) It should be "Compare with _**Singleton**_ and WPE" 
2) I would add (before the phrase above) "KME is a part of the worker pool responsible for the key management. It has access to the worker's private keys and controls workorder execution by WPE"